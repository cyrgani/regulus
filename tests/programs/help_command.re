import(help),
=(x, help(doc)),
assert_eq(x, null),
