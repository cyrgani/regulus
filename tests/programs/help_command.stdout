 Returns the documentation string for a function.
 TODO: implement this for non-builtin functions as well.
 TODO: also show the argc somehow
 TODO: consider printing directly too
 TODO: consider removing the leading newline (should then be done in `functions!`) 
